kessler/kessler.F90

```diff
@@ -14,17 +14,19 @@ module kessler
    real(kind_phys) :: cp    ! heat capacity at constant pressure, J/(kgK)
    real(kind_phys) :: lv    ! latent heat of vaporization, J/kg
    real(kind_phys) :: psl   ! reference pressure at sea level, mb
+   real(kind_phys) :: rair  ! dry air gas constant J/(kgK)
```

Comment:
Instead of `rair`, why not replace it with `inv_kappa` or something similar, calcualte it in `kessler_init` and then use it in `kessler_run` instead of `xk`? You have no other use of `rair`.